ibridges/search.py

```diff
@@ -60,17 +60,17 @@ def search_data(session: Session, path: Optional[Union[str, IrodsPath]] = None,
     # one data search in case path is a collection path and we want to retrieve all data there
     # one in case the path is or ends with a file name
     if path:
-        path = IrodsPath(session, path)
-        parent = path.parent
-        name = path.name
+        path = str(path)
+        parent = path.rsplit("/", maxsplit=1)[0]
```

Comment:
What happens if the path is simply "/"?

Reply:
If you do not have access to read the root collection there are no results. For admins this will list the content directly under the root collection. With "/%" as irods admin you will receive a time out since it takes too, long to list the whole iRODS server.